<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login_Window</class>
 <widget class="QDialog" name="Login_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(148, 143, 124)</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(15, 86, 90);
color: rgb(255, 255, 255);
font-size: 30px;
border-radius: 10px</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; color:#ffffff;&quot;&gt;Username&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(15, 86, 90);
color: rgb(255, 255, 255);
font-size: 30px;
border-radius: 10px</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; color:#ffffff;&quot;&gt;Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
	background-color: rgb(23, 137, 143);
	color: rgb(255, 255, 255);
	font-size: 30px;
	border-radius: 30px
}
QPushButton:pressed{
	background-color:rgb(14, 85, 89);
	color: rgb(255, 255, 255);
	font-size: 30px;
	border-radius: 30px
}
QPushButton{
background-color: rgb(15, 86, 90);
color: rgb(255, 255, 255);
font-size: 30px;
border-radius: 30px
}
</string>
   </property>
   <property name="text">
    <string>LogIn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registration_button">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
	background-color: rgb(23, 137, 143);
	color: rgb(255, 255, 255);
	font-size: 20px;
	border-radius: 20px
}
QPushButton:pressed{
	background-color:rgb(14, 85, 89);
	color: rgb(255, 255, 255);
	font-size: 20px;
	border-radius: 20px
}
QPushButton{
background-color: rgb(15, 86, 90);
color: rgb(255, 255, 255);
font-size: 20px;
border-radius: 20px
}
</string>
   </property>
   <property name="text">
    <string>Registration</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
